### ISSUE TITLE
Custom repeat days

### ISSUE THREAD

**Issue by hesyifei** [msherry/ical2org#6]
For example, my ics has this rule repeating an event on Monday, Wednesday, and Friday.
```
RRULE:FREQ=WEEKLY;UNTIL=20211204T075959Z;INTERVAL=1;BYDAY=MO,WE,FR
```

However, after converting, it only has one day left:
```
<2021-09-22 Wed 16:00-17:00 +1w>
```

It would be great if these repeating rules are supported. Thanks!

**Comment by msherry** [comment]
I agree, that would be nice! I'll try to get to this when I can, but if you want to try to put a pull request together, I'd be happy to review it!

**Comment by szabolcs-szilagyi** [comment]
hello @msherry,

I've created a PR for this topic, once you have time please have a peak. Not the most elegant solution, but with my awk skills.. :smile:
